dotnet/Service/Program.cs

```diff
@@ -38,11 +38,12 @@
 
 // Inject memory client and its dependencies
 // Note: pass the current service collection to the builder, in order to start the pipeline handlers
-ISemanticMemoryClient memory = new MemoryClientBuilder(appBuilder.Services).FromAppSettings().Build();
+var memoryClientBuilder = new MemoryClientBuilder(appBuilder).FromAppSettings();
```

Comment:
Has this service been deployed outside of dev-box? (I think it'd also be susceptible to the 503 error.)